wait/host_port.go

```diff
@@ -171,7 +195,6 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 			return errors.New("/bin/sh command not executable")
```

Comment:
Wdyt if we create an error type for the 126 exit code, and we check for that error in L140 (https://github.com/testcontainers/testcontainers-go/pull/1039/files#diff-ad6cdf7fcae968ad7307fe3e8717bcbc7e43c461424e76ca3953cd215ac9395bR140), so that we always automatically detect the `/bin/sh` command is not executable? It will cause removing the new `internalCheckDisabled` fields/funcs, and will be transparent for the users. Probably a log warn?

Reply:
I've updated that and I am now logging. I couldn't see any sort of structured logging library in use, but I may have missed it.

Reply:
You could use the default Logger instance from https://github.com/testcontainers/testcontainers-go/blob/main/logger.go#L13